Tests/Singulink.Globalization.Currency.Tests/ImmutableSortedMoneySetTests/RemoveTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using Shouldly;
+
+namespace Singulink.Globalization.Tests.ImmutableSortedMoneySetTests;
+
+[TestClass]
+public class RemoveTests
+{
+    private static readonly Money Usd100 = new(100m, "USD");
+    private static readonly Money Cad50 = new(50m, "CAD");
+    private static readonly Money Eur25 = new(25m, "EUR");
+    private static readonly ImmutableSortedMoneySet _immutableSet = [Usd100, Cad50, Eur25];
```

Comment:
Immutable/constant `static readonly` fields should be cased like `ImmutableSet` instead of `_immutableSet`. In the interest of brevity and consistency, I'd like to keep the "set under test" name as just the appropriately-cased version of the word "set" for these tests, so rename the field to `Set`.